### ISSUE TITLE
borrowing code

### ISSUE THREAD

**Issue by sandsmark** [glouw/openempires#5]
Hi!

Seems like I wasn't the only one who found openage a bit too complex (deps-wise and code-wise), I also started my own thing; https://github.com/sandsmark/freeaoe (depending only on sfml).

Not really a bug, but I wonder how comfortable you are if I just borrow code outright from you. We use the same license, but I don't want to create any animosity. :-)

(If I get tired of c++ I might just start working on this instead anyways.)

**Comment by glouw** [comment]
We share a common goal - you are more than welcome to use whatever you need. 

I plan to have openempires be a multiplayer only client, and to be done v1.0 in april. Maybe we can unify our efforts after April? All the code you'll find in openempires is written in a dialect that is compilable out of the box with C11 / C++11 for the very reason such as projects like your Free AOE.

Regards!

PS. I highly believe in parallel implementations: openempires, freeaoe, openage, we all learn something special on our own

**Comment by sandsmark** [comment]
I only started on freeaoe because I wanted to play aoe2 on Linux, fixing the GDI implementation in Wine seemed almost impossible, and I couldn't wrap my head around the openage code (I tried to work on that first).

And I agree with implementation in parallell, initially I started on freeaoe just to figure out and test stuff to pull back into openage (originally just the unit icons, and then it kind of snowballed).

And yeah, it seems like openempires and freeaoe are very complementary, I just focused on getting something playable in single player ASAP (so I got the campaign/scenario support up and running first, then started on the AI stuff, but then got sidetracked trying to figure out the savegames...).

Unfortunately I also used freeaoe as an excuse to test some of the new c++ stuff (up to c++20), so porting to C is not going to be trivial.
